This file is part of MXE. See LICENSE.md for licensing information.

Enable loop points when using libmodplug.

Note SDL_mixer r826 "Fixed bug 1558 - mikmod broken volume and loop support"
enabled loop points in music_mikmod.c, but not in music_modplug.c

--- a/src/codecs/music_modplug.c        Mon Jan 11 05:55:02 2021 +0300
+++ b/src/codecs/music_modplug.c        Mon Jan 25 23:06:20 2021 +1300
@@ -152,7 +152,7 @@
     settings.mBassRange = 50;
     settings.mSurroundDepth = 0;
     settings.mSurroundDelay = 10;
-    settings.mLoopCount = 0;
+    settings.mLoopCount = -1;
     modplug.ModPlug_SetSettings(&settings);
     return 0;
 }
